examples/fieldenum/README.md

```diff
@@ -0,0 +1,16 @@
+Field Number Enumerations
```

Comment:
Could you please add same info in doxygen format to:
src/generator/generatorbase.h
That will give us this info in API reference as well. Thank you!